### PR TITLE
Implement account file format standard

### PR DESCRIPTION
**Pull request by Thykof**

### CODE REVIEW

**Review by thomas-senechal — CHANGES_REQUESTED** [review]

File: pkg/wallet/wallet.go [hunk 1]
```diff
@@ -48,6 +50,46 @@ type Wallet struct {
 	KeyPair  KeyPair
 }
 
+type AccountSerialized struct {
+	Version      uint8
```
Comment (mazmassa) [inline pkg/wallet/wallet.go:54]: Shouldn't Version be in string format to follow the semantic versioning!? (1.2.3 - 2.2.2, etc) 🤔 :

MAJOR version when you make incompatible API changes
MINOR version when you add functionality in a backwards compatible manner
PATCH version when you make backwards compatible bug fixes

Comment (thomas-senechal) [reply pkg/wallet/wallet.go:54]: Furthermore, it's following the Wallet file format standard defined [here](https://github.com/massalabs/massa-standards/blob/main/wallet/file-format.md)

File: pkg/wallet/wallet.go [hunk 2]
```diff
@@ -48,6 +50,46 @@ type Wallet struct {
...
+	Nickname     string
+	Address      string
+	Salt         [16]byte
+	Nonce        [12]byte
+	CipheredData []byte
+	PublicKey    []byte
+}
+
+func (accountSerialized *AccountSerialized) ToAccount() Wallet {
```
[the remark this answers is not in the document]
Comment (Thykof) [reply pkg/wallet/wallet.go:63]: yes but I don't know if this PR is the good moment to do it.

**Review by ASAPSegfault — APPROVED** [review]

File: pkg/wallet/wallet.go (hunk 1 above)
Comment (mazmassa) [reply pkg/wallet/wallet.go:54]: No worries, folks! It's up to you!

It was just a suggestion! Closing...

Comment (mazmassa) [reply pkg/wallet/wallet.go:54]: Ops... was already closed! 😞

**Review by thomas-senechal — APPROVED** [review]

**Pull request merged**